plugins/modules/inventory_intent.py

```diff
@@ -40,7 +40,7 @@
     required: True
     suboptions:
       cli_transport:
-        description: Device's cli transport protocol. Required for Adding Network Devices.
+        description: Device's cli transport protocol(ssh/telnet). Required for Adding Network Devices.
```

Comment:
Can we say like below?
To add Network Devices, it's important to mention the way the device communicates, which can be either through SSH or Telnet.
(or)
The essential prerequisite for adding Network Devices is the specification of the transport protocol (either SSH or Telnet) used by the device

Reply:
added.